_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
+---
+
+What is Neovim?
+---------------
+
+Neovim is a fork of the venerable text-editor vim, focused on extensibility and
+usability. It is not a rewrite but a continuation and extension of Vim. Many
+clones and derivatives exist, some very clever—but none are Vim. Neovim is built
+for users who want the good parts of Vim, and more. See a list of differences
+via [:help vim-differences](https://neovim.io/doc/user/vim_diff.html)
+
+### Participating
+
+If you are interested in contributing to neovim, visit [the github
+page](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md) for details.
+We have a great test suite and automated CI, so you can just jump in and have a
+go. Don't be afraid to open a PR to get comments or just take advantage of the
+infrastructure!
+
+To get in touch with the team, join the [gitter channel](gitter.im/neovim) (also
+accessible via [IRC](https://webchat.freenode.net/?channels=neovim)), or visit
+the [neovim subreddit](https://www.reddit.com/r/neovim/), which many of the
+developers read. Taking part in conversations is a contribution, too!
+
+
+What's new
+----------
+
+The [last newsletter](https://neovim.io/news/2016/11/) has gathered some dust,
+so we want to bring you up-to-date in the neovim world. The elephant in the room
+is obviously
+
+### Release 0.2
+
+It features windows support, various terminal improvements, full API
+documentation for developers and a host of smaller fixes and improvements.  See
+the [release page](https://github.com/neovim/neovim/releases/tag/v0.2.0) for a
+detailed summary and download opportunities.
+
+### 0.2.1-dev for Users
+- Window specific background (https://github.com/neovim/neovim/pull/6597)
+- Windows TUI (https://github.com/neovim/neovim/pull/6315)
+- AppImage packages for Linux (https://github.com/neovim/neovim/pull/6638)
+
+### 0.2.1-dev for Developers
+- Static Analysis with PVS (???)
+- Externalized tabline (???)
+- Eval.c refactor (https://github.com/neovim/neovim/pull/5119)
+- ui_events now reported in api metadata (???)
+- First usage of lua to implement core functionality
+(https://github.com/neovim/neovim/pull/67890)
```

Comment:
This link is wrong. https://github.com/neovim/neovim/pull/6789 maybe?

Reply:
Fixed, thanks.